_layouts/default.html

```diff
@@ -11,6 +11,8 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Roboto+Mono:400|Roboto:400,700" rel="stylesheet">
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
     <link rel="icon" href="/assets/images/favicon.png" type="image/png">
+    <script src='https://js.tito.io/v1' async></script>
```

Comment:
can it be loaded within body to not block the site while loading?

Reply:
Isn't it loaded asynchron anyway?